docs/compendium/public_share.md

```diff
@@ -1,6 +1,6 @@
 # Public share
 
-Upload an unvalidated research compendium by submitting a link to a cloud resource. Currently, only sciebo (https://www.sciebo.de/en/) is supported.
+Upload an unvalidated research compendium by submitting a link to a cloud resource. Currently, sciebo (https://www.sciebo.de/en/) and Zenodo (https://sandbox.zenodo.org) is supported.
```

Comment:
We support both sandbox.zenodo.org and zenodo.org, right? As for the case "which one to use if only the zenodo id is given", that can be detailed below. Here, please mention both.